src/Common/Constants.ts

```diff
@@ -353,7 +355,8 @@ export const StyleConstants = require("less-vars-loader!../../less/Common/Consta
 
 export class Notebook {
   public static readonly defaultBasePath = "./notebooks";
-  public static readonly heartbeatDelayMs = 5000;
+  public static readonly heartbeatDelayMs = 60000;
+  public static readonly initialHeartbeatDelayMs = 5000;
```

Comment:
>initialHeartbeatDelayMs [](http://example.com/codeflow?start=25&length=23)

why do we need a special initial heartbeat delay?

Reply:
Previously we have memory heart beat with 5 sec, so initial heart beat also use to start after 5 sec. But now as we changed it to 60 secs, so need to initiate a memory call after allocation. Made an explicit call in allocate.